frontend/src/components/login/loginstyle.css

```diff
@@ -11,17 +8,23 @@
     align-items: center;
     height: 100%;
     overflow: auto;
+    color: 'white';
+    background: radial-gradient(circle at top, #000000, #1b1534, #4d217a, #643082);
```

Comment:
Same here: why are we changing styling/colors?
Isn't there an extra branch for that?

Reply:
I think these changes are shown as we tried to resolve the merge conflicts